app/models/invite.py

```diff
@@ -0,0 +1,80 @@
+import requests
+import os
+import hashlib
+
+from random import random
+
+from django.db import models
+from django.core import mail
+from django.core.urlresolvers import reverse
+from django.core.mail import EmailMessage
+
+from .project import Project
+from .user import UserAuthentication
+
+
+class ProjectInvite(models.Model):
+	"""Project invite model."""
+	email = models.EmailField(blank=False)
+	invite_code = models.CharField(primary_key=True, max_length=100, blank=False)
+	project = models.ForeignKey(Project)
+
+	PENDING = 0
+	ACCEPTED = 1
+	REJECTED = 2
+	ACCEPT_CHOICES = (
+		(PENDING, 'Pending'),
+		(ACCEPTED, 'Accepted'),
+		(REJECTED, 'Rejected'),
+	)
+	accept = models.PositiveSmallIntegerField(
+		choices=ACCEPT_CHOICES,
+		default=PENDING
+	)
+	uid = models.ForeignKey(UserAuthentication)
+
+	def send_invite_email(self):
+		"""Method to send emails (containing project invite codes)
+			to invitees through email.
+		"""
+		connection = mail.get_connection()
+
+		# set the invite code to the ProjectInvite object
+		self.invite_code = self.create_invite_code()
+
+		host = os.environ.get('LIMBER_HOST')
+
+		url = 'http://' + host + reverse('project-invites-list') + self.invite_code
+
+		body = """Hi there!\n
```

Comment:
Later on, let's have the messages pulled from a table in the DB / or config file.